engine/maptile.py

```diff
@@ -38,3 +38,11 @@ def reveal_npcs(self):
     
     def greet_npcs(self):
         return [npc.greet() for npc in self.local_npcs]
+
+    def pickupitem(self, description):
```

Comment:
Let's try to either use underscores or CamelCase in function names to keep up readability. Which do you prefer?
PickUpItem, or pick_up_item ?